pytoda/datasets/annotated_dataset.py

```diff
@@ -0,0 +1,92 @@
+"""Implementation of AnnotatedDataset class."""
+import torch
+import pandas as pd
+from torch.utils.data import Dataset
+from ..types import DrugSensitivityData
+
+
+class AnnotatedDataset(Dataset):
+    """
+    Annotated dataset implementation.
+    """
+
+    def __init__(
+        self,
+        annotations_filepath: str,
+        input_data: Dataset,
```

Comment:
Remove `backend` since it's unsed. I would rename `input_data` to `dataset` for consistency.

Reply:
both done